microsoft-edge/devtools-guide-chromium/accessibility/preferred-color-scheme-simulation.md

```diff
@@ -10,35 +10,34 @@ keywords: microsoft edge, web development, f12 tools, devtools
 ---
 # Dark or light color scheme simulation  
 
```

Comment:
Needs a clarifying point explicitly up front: Is this article about changing the theme of the DevTools portion of the window, or changing the rendering of the webpage?  ( = the Appearance commands vs the Rendering commands in the Command Menu.)